corfudb-tools/src/main/java/org/corfudb/browser/CorfuStoreBrowser.java

```diff
@@ -65,6 +73,20 @@
     private final CorfuRuntime runtime;
     private final String diskPath;
 
+    private final ConcurrentMap<String, DescriptorProtos.FileDescriptorProto>
+        fdProtoMap = new ConcurrentHashMap<>();
+    /*
+     * Maps the Message name to the name of the FileDescriptorProto containing it.
+     */
+    private final ConcurrentMap<String, String> messagesFdProtoNameMap =
```

Comment:
Isn't this the same map in DynamicProtobufSerializer class which can be used instead?

Reply:
thats right.  I wasnt sure if we wanted to expose anything from DynamicProtobufSerializer, so replicated it.  I will the change to reuse methods/variables from there wherever possible.